src/main/java/org/jabref/gui/exporter/ExportToClipboardAction.java

```diff
@@ -60,12 +61,18 @@ public void execute() {
                                                         .sorted(Comparator.comparing(Exporter::getName))
                                                         .collect(Collectors.toList());
 
+        //Find default choice, if any
+        Exporter defaultChoice = exporters.stream()
+                .filter(exporter -> exporter.getName().equals(Globals.prefs.get(JabRefPreferences.LAST_USED_CLIPBOARD_EXPORT)))
+                .findAny()
+                .orElse(null);
+
         Optional<Exporter> selectedExporter = dialogService.showChoiceDialogAndWait(Localization.lang("Export"), Localization.lang("Select export format"),
-                                                                                    Localization.lang("Export"), exporters);
+                    Localization.lang("Export"),defaultChoice, exporters);
```

Comment:
Please insert a space:
```suggestion
                    Localization.lang("Export"), defaultChoice, exporters);
```